hal/src/nRF52840/lwip/lwip_logging.cpp

```diff
@@ -33,3 +47,21 @@ void lwip_log_message(const char* fmt, ...) {
     log_message_v(1, "lwip", &attr, nullptr /* reserved */, tmp, args);
     va_end(args);
 }
+
+void ppp_dbglog(const char *fmt, ...) {
+    // Only enable PPP negotiation packet logs
+    if (strstr(fmt, "%P") == nullptr) {
+        return;
+    }
+
+    va_list args;
+
+    va_start(args, fmt);
+    char tmp[LOG_MAX_STRING_LENGTH] = {};
+    ppp_vslprintf(tmp, sizeof(tmp) - 1, fmt, args);
+    va_end(args);
+    tmp[strcspn(tmp, "\r\n")] = 0;
+
+    LogAttributes attr = {};
+    log_message(LOG_LEVEL_TRACE, "lwip.ppp", &attr, nullptr /* reserved */, tmp);
```

Comment:
I'm assuming we arent enabling lwip logging in RTK platform as well just for interest of time / not needed so far?